python/hsfs/engine/python.py

```diff
@@ -1510,7 +1510,7 @@ def _write_dataframe_kafka(
                 topic_name=feature_group._online_topic_name,
                 feature_store_id=feature_group.feature_store_id,
                 offline_write_options=offline_write_options,
-                high=True,
+                high=False,
```

Comment:
Not knowing this code, what does this change do?

Reply:
`kafka_get_offsets` get offsets for specific topic, the results would look something like this: `topic_name,0:1,1:5,2:6,3:3,4:10`.
Setting `high` to true makes sure we get latest/highest offsets, while setting it to false we get earliest/smallest offsets. Previously it was correct, but it looks like the value was unintentionally changed here: https://github.com/logicalclocks/hopsworks-api/commit/834d13ff1c0b21c6b852ea4fc60397901124cd01#diff-80e02e05a9d4246ee3bdd7a91b698057815fa04a2234772c883aba8b8770f977L1402